apps/admin/frontend/package.json

```diff
@@ -125,10 +125,9 @@
   },
   "devDependencies": {
     "@jest/types": "^29.6.1",
-    "@testing-library/jest-dom": "^5.16.4",
-    "@testing-library/react": "^12.1.5",
-    "@testing-library/react-hooks": "^8.0.1",
```

Comment:
`@testing-library/react-hooks` has been absorbed by `@testing-library/react`, and you'll see it being replaced in tests in subsequent commits.